src/streamlink/plugins/nbcnews.py

```diff
@@ -28,18 +52,14 @@ def can_handle_url(cls, url):
     def get_title(self):
         return 'NBC News Now'
 
-    @Plugin.broken(3123)
     def _get_streams(self):
-        html = self.session.http.get(self.url).text
-        match = self.js_re.search(html)
-        js = self.session.http.get(match.group(0)).text
-        match = self.api_re.search(js)
-        log.debug('API ID: {0}'.format(match.group(1)))
-        api_url = self.api_url.format(match.group(1))
+        id = self.session.http.get(self.url, schema=self.json_data_schema)
```

Comment:
don't use a Built-in Function name https://docs.python.org/3.6/library/functions.html#id

Reply:
Oh, I hadn't realised that, thanks.